cosmo_tester/test_suites/test_snapshots/single_manager.py

```diff
@@ -0,0 +1,69 @@
+import time
+from cosmo_tester.test_suites.test_blueprints.nodecellar_test import \
+    OpenStackNodeCellarTestBase
+from cloudify_rest_client.executions import Execution
+
+
+class SingleManagerTest(OpenStackNodeCellarTestBase):
+    """
+    This test deploys nodecellar, creates an additional deployment and a
+    snapshot, deletes both deployments and the corresponding blueprint,
+    restores the snapshot and validates that the manager is consistent
+    and operational - uninstalling nodecellar, deleting both deployments
+    and deleting the blueprint should succeed.
+    """
+
```

Comment:
You mean this line?

self.client.snapshots.create(snapshot_id, True, True)

I also thought about passing various False/True combinations here, as separate test's variants, but decided that maybe a "full option" is enough..